### PR TITLE
UTOPIA-236: Know what what the next steps are .. complexity, time, signatures 

### PR DESCRIPTION
**Pull request by Bottle7**
# Description

This PR includes the following proposed change(s):

- Creates Card component
- Refactors Callout into its own component file
- Creates PPQResults molecular component
- Adjusts existing styling to better conform to designs
- Adds SASS files for each new component for ease of reading (bootstrap not working as intended atm)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Rendered individual components to the landing page during development and finally the PPQResults component to the landing page to test spacing

## Development Dependency Working Agreement
- [ ] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [x] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [x] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

### CODE REVIEW

File: src/frontend/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "@fortawesome/free-solid-svg-icons": "6.2.0",
     "@fortawesome/react-fontawesome": "0.2.0",
     "breakpoint-sass": "3.0.0",
+    "esbuild-darwin-arm64": "0.15.11",
```
Comment (Bottle7) [inline src/frontend/package.json:27]: This was added automatically because I use macOS and is for bundling TypeScript.

**Comment by binggaofw** [comment]
is the lint-staged func still not work in your local ? looks the lint does not work for all the files

**Comment by Bottle7** [comment]
@binggaofw Yes, I have to --no-verify to commit code right now because the linter throws an error on the first line of every file.

**Comment by Bottle7** [comment]
The lint issue has been remedied in a recent change.

**Review by binggaofw — APPROVED** [review]

**Pull request merged**